zebra-rpc/src/methods.rs

```diff
@@ -273,8 +273,15 @@ where
     {
         let runner = Queue::start();
 
+        let mut app_version = app_version.to_string();
+
+        // Match zcashd's version format, if the version string has anything in it
+        if !app_version.is_empty() && !app_version.starts_with('v') {
+            app_version.insert(0, 'v');
```

Comment:
I don't think we need to do this. Is lightwalletd expecting the prefix?

A version of zcash will never match a zebra one anyways.

Reply:
I don't particularly mind either way, but this is why I changed it:

Lightwalletd documents that `build` starts with a "v":
https://github.com/zcash/lightwalletd/blob/83bb19918fb36a5f6d5013fba7938daf3d73db1d/walletrpc/service.proto#L71

This version is sent to wallets and might be displayed to the user. So using a different format might cause wallet bugs or user confusion.

What do you think?

Reply:
Fair enough to me.

Reply:
Feel free to resolve at your convenience so it can get merged.